packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -118,7 +121,8 @@ export class MirrorNodeClient {
             headers: {
                 'Content-Type': 'application/json'
             },
-            timeout: 10 * 1000
+            signal: controller.signal,
```

Comment:
Should we be doing this per client or per request?
As in here in the create or in the get?

Reply:
Refactored to implement in the config defined in the request method.